packages/core/src/database/postgres/service.ts

```diff
@@ -244,9 +247,9 @@ export class PostgresDatabaseService implements BaseDatabaseService {
         functionId: m.function_id,
         functionName: m.function_name,
         fromCheckpoint: m.from_checkpoint
-          ? decodeCheckpoint(m.from_checkpoint)
+          ? decodeCheckpoint(m.from_checkpoint.toString())
           : null,
-        toCheckpoint: decodeCheckpoint(m.to_checkpoint),
+        toCheckpoint: decodeCheckpoint(m.to_checkpoint.toString()),
```

Comment:
Are these still necessary now that we're using VARCHAR?

Reply:
good call forgot to remove these!